### DISCUSSION TITLE
Interesting related

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#694] [Announcements]
## oss
- https://github.com/solan-solan/HeightMap/tree/smooth_lod_passing
- https://github.com/wzhengsen/StarryX
- https://github.com/aismann/SimpleSnake
- https://github.com/EugenyN/TanksKombat
- https://github.com/aismann/game-controller-test
- https://www.codeandweb.com/texturepacker/tutorials/axmol
- https://github.com/DelinWorks/arabic-text-renderer-cpp-axmol
- https://github.com/CodeAndWeb/axmol-sprite-sheet-tutorial
- https://github.com/CodeAndWeb/axmol-game-engine-physics-tutorial
- https://www.codeandweb.com/spriteilluminator/tutorials/how-to-use-light-effects-with-axmol-game-engine
- https://github.com/Xrysnow/cocos-x
- https://github.com/aismann/axmol_examples
- https://github.com/Open-GD/OpenGD
- https://github.com/aismann/SpriterPro4axmol
- https://github.com/aesophor/vigilante
- https://github.com/nowasm/axmol (wasm support, already merged into main repo)
- https://github.com/DelinWorks/Rebound
- https://github.com/danialias/InfiniteScrollNode
- https://github.com/paulocoutinhox/axmol-extensions
- https://github.com/danialias/LocalizedString
- https://github.com/OscarLeif/FlappyBird-Axmol
- https://github.com/ChilliHugger/The-Lords-Of-Midnight
- https://github.com/johnspeny/hitbubble
- https://github.com/rh101/PostEffectDemo
- https://github.com/remtori/axmol-js (WIP)
- https://github.com/kuroppoi/opendw

## products

- https://paulocoutinho.pages.dev/games
- 末世部落（wasm）：https://h5.7k7k.com/mb/mb2/75b088dcc5a6a5a09d5d17b333002507.html?gid=92bfb4b52977006e46677bc05a825f12&tid=95170&qs=1
- https://www.anagramarena.com/
- https://cuttercards.site/
- Hello Seafood 2
  - Google Play: https://play.google.com/store/apps/details?id=com.appstree.helloseafood2
  - Apple Store: https://apps.apple.com/app/id958009330
- Hello Coffee Shop
  - Google Play: https://play.google.com/store/apps/details?id=com.appstree.hellocoffeeshop
  - Apple Store: https://apps.apple.com/app/id529504526
- https://store.steampowered.com/app/2443900/A_Simple_Expurriment/

## videos

- https://www.youtube.com/watch?v=saTY9peU9IY
- https://www.youtube.com/watch?v=arbdwd84NEQ
- https://www.bilibili.com/video/BV1Sh4y1v7UZ/?spm_id_from=333.337.search-card.all.click&vd_source=a353adb24710bf5cec1234fe8f6f37a4
- 2d platformer showcase: https://www.youtube.com/watch?v=3nPVrxiO-bc

**Comment by aismann** [comment]


**Reply by halx99** [reply]
done.

**Reply by aismann** [reply]
> done.
@halx99 
History can be removed again. link is on the page above

**Comment by DelinWorks** [comment]
@halx99 could you add https://github.com/Open-GD/OpenGD ?

**Reply by halx99** [reply]
done

**Comment by danialias** [comment]
Maybe you can consider adding https://github.com/danialias/LocalizedString to the list?

**Comment by halx99** [comment]
> Maybe you can consider adding https://github.com/danialias/LocalizedString to the list?

done

**Comment by paulocoutinhox** [comment]
Mine is https://paulocoutinho.pages.dev/games instead of https://paulocoutinho.pages.dev/apps. Thanks.

**Reply by halx99** [reply]
done

**Comment by GamesStudioTwo** [comment]
Hi there do you want to add this video as its a more deeper dive in to the engine and  remove other as this show the engine in more better light
https://www.youtube.com/watch?v=saTY9peU9IY

**Reply by halx99** [reply]
done

**Comment by ribachgames** [comment]
Hi @halx99, would you consider adding https://www.anagramarena.com/ to the list of games?

**Reply by halx99** [reply]
> Hi @halx99, would you consider adding https://www.anagramarena.com/ to the list of games?

done

**Reply by ribachgames** [reply]
Hi @halx99, could you also add the same link to [Made in Axmol](https://github.com/axmolengine/axmol/wiki/Made-in-Axmol) showcase page?

**Reply by halx99** [reply]
> Hi @halx99, could you also add the same link to [Made in Axmol](https://github.com/axmolengine/axmol/wiki/Made-in-Axmol) showcase page?

done

**Comment by johnspeny** [comment]
I have been working on this cards game and now ready for the public. Could you consider adding it to list of games made with axmol.

https://cuttercards.site/

**Reply by aismann** [reply]
> I have been working on this cards game and now ready for the public. Could you consider adding it to list of games made with axmol.
> 
> https://cuttercards.site/

Done.
